README.rst

```diff
@@ -15,10 +15,9 @@ architectural style, and tries to do as little as possible while
 remaining highly effective.
 
 Falcon apps work with any WSGI server, and run great under
-CPython 2.6-2.7, PyPy 2.7, Jython 2.7, and CPython 3.3+.
+CPython 2.7, PyPy 2.7, Jython 2.7, and CPython 3.3+.
 
-(Note: Support for CPython 2.6 and Jython 2.7 is deprecated and will be
-removed in Falcon 2.0.)
+(Note: Support for Jython 2.7 is deprecated and will be removed in Falcon 2.0.)
```

Comment:
Given that 2.7 is still the current version of Jython, I would recommend changing __Jython 2.7__ to __Jython__.

Reply:
@cclauss Done!